projects/demo/src/app/pages/modal-demo/modal-demo.component.ts

```diff
@@ -41,20 +43,51 @@ export class ModalDemoComponent {
     TS: `${BASE_REQUEST_PATH}/modal-demo-example-4/modal-demo-example-4-helper/modal-demo-example-4-helper.component.ts`,
   };
 
+  public readonly example5Content: Record<string, string> = {
+    HTML: `${BASE_REQUEST_PATH}/modal-demo-example-5/modal-content/modal-demo-example-5.component.html`,
+    SCSS: `${BASE_REQUEST_PATH}/modal-demo-example-5/modal-content/modal-demo-example-5.component.scss`,
+    TS: `${BASE_REQUEST_PATH}/modal-demo-example-5/modal-demo-example-5-helper/modal-demo-example-5-helper.component.ts`,
+  };
+
   public readonly formGroup: FormGroup = new FormGroup({
     hasBackdrop: new FormControl(true),
     closeOnBackdropClick: new FormControl(true),
     isBackdropTransparent: new FormControl(false),
     isFullscreen: new FormControl(false),
+    height: new FormControl(''),
+    size: new FormControl(''),
   });
 
+  public readonly sizeOptions: PropsOption[] = [
+    {
+      caption: 'empty',
+      value: '',
+    },
+    {
+      caption: 'XS',
+      value: 'xs',
+    },
+    {
+      caption: 'S',
+      value: 's',
+    },
+    {
+      caption: 'M',
+      value: 'm',
+    },
+    {
+      caption: 'L',
+      value: 'l',
+    },
+  ];
+
   constructor(private readonly modalsService: ModalsService, private readonly injector: Injector) {}
 
   public openModal(): void {
     const openedModal: OpenedModal<string> = this.modalsService.open(ModalDemoContentComponent, {
       ...this.formGroup.value,
       injector: this.injector,
-      height: 400,
+      height: isNil(this.formGroup.value.size) ? 400 : null,
```

Comment:
```suggestion
      height: isEmpty(this.formGroup.value.size) ? 400 : null,
```
`'' === null` или `'' === undefined` будет false, а начальное значение контрола - ''